src/components/events/GoalItem/index.js

```diff
@@ -0,0 +1,28 @@
+var React = require('react');
+
+module.exports = React.createClass({
+  displayName: 'CampaignGoalItem',
+
+  propTypes: {
+    name: React.PropTypes.string.isRequired,
+    goal: React.PropTypes.number.isRequired,
+    count: React.PropTypes.number.isRequired
+  },
+
+  render: function() {
+    var percentage = this.props.count / this.props.goal * 100;
+
+    if (percentage > 100) {
```

Comment:
Even slightly more robust:

```
Math.max(Math.min(this.props.count / this.props.goal * 100, 100), 0)
```

Protects against negative values for count / goal